cmd/outputs.go

```diff
@@ -92,9 +94,15 @@ func createOutputs(
 		RuntimeOptions: test.preInitState.RuntimeOptions,
 		ExecutionPlan:  executionPlan,
 	}
-	result := make([]output.Output, 0, len(test.derivedConfig.Out))
 
-	for _, outputFullArg := range test.derivedConfig.Out {
+	outputs := test.derivedConfig.Out
+	if test.derivedConfig.WebDashboard.Bool {
+		outputs = append(outputs, dashboard.OutputName)
+	}
+
+	result := make([]output.Output, 0, len(outputs))
+
+	for _, outputFullArg := range outputs {
```

Comment:
Looking at this and the changes to the way we format stuff ... I wonder ... why?

Like why do you want ot have an additional `--web-dashboard` cli flag when `-o dashboard` ... works.

Why do you want to move it on separate line when outputs already have a way to show a url for users to use as in 
![image](https://github.com/grafana/k6/assets/312246/99d57f7a-1586-4528-9618-5a9b3286d02f)
^ arguably the above can be improved, but I see no reason to move it in a different one.

And this will practically limit the integration to the above "registration" of the output.

Reply:
> Like why do you want ot have an additional `--web-dashboard` cli flag when `-o dashboard` ... works.

Because this is what is written in the design document, if you want to re-discuss it open the question to the working group. It's perfectly fine for me if it has to be activated with the -o flag.

Reply:
As @szkiba mentioned, this was a UX/product decision made by the workgroup that is reflected in the design document. 

Part of the reasoning behind that decision was a desire to have the dashboard become a first-class citizen of the k6 UX down the road, the fact that it acts as an output under the hood being judged an implementation detail.

If there is a strong **technical** argument to be made to keep exposing/displaying it as an output as part of our command-line UI, we'd be happy to discuss it at the beginning of next year when everyone is back 🤝

Reply:
> first-class citizen of the k6 UX 

Outputs are first class citizen.

> down the road

Kind of implies to me this shouldn't be in the initial release.

The technical and none technical argument is the same - users already have the concept of output. Adding a thing that is clearly and output and then treating it differently isn't cohesive. 

At best people will be confused why there are two ways to the same thing at worse they will think they are two different things. And anyone who works on it will need to wonder the same thing.

On connected note - yes the current summary fits this same bill, except that it predates most of that, and if possible I will certainly try to push it towards being an output through and through.

But adding more special cases IMO doesn't help anyone.

Reply:
I see your point 👍🏻 I personally don't have a strong opinion on this.

Will add it to the agenda of our internal discussions when everyone is back from 🎄 holidays, and I'll invite you to it so we can decide on a way forward that makes sense for everyone 🙇🏻

Reply:
We've discussed this internally, and our preference goes with proceeding with the current UX provided by the PR:
* The feature needs to be activated through an environment variable
* when activated, it has a dedicated line in the output to point users to the URL
* the `--output` argument is supported for consistency and to align with a larger idea to mark outputs (including the end-of-test summary), more explicitly as such in the future.

```
K6_WEB_DASHBOARD=true ./k6 run ../extensions/xk6-dashboard/script.js

          /\      |‾‾| /‾‾/   /‾‾/
     /\  /  \     |  |/  /   /  /
    /  \/    \    |     (   /   ‾‾\
   /          \   |  |\  \ |  (‾)  |
  / __________ \  |__| \__\ \_____/ .io

     execution: local
        script: ../extensions/xk6-dashboard/script.js
 web dashboard: http://127.0.0.1:5665
        output: -

     scenarios: (100.00%) 2 scenarios, 9 max VUs, 12m30s max duration (incl. graceful stop):
              * camel: Up to 5 looping VUs for 12m0s over 6 stages (gracefulRampDown: 0s, gracefulStop: 30s)
              * snake: Up to 4 looping VUs for 12m0s over 12 stages (gracefulRampDown: 0s, gracefulStop: 30s)
```